### ISSUE TITLE
The inputTransformation listener should not insert the content

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-clipboard#31]
The behaviour of this event should follow its name – transform the content. The rest should be in the caller's hands.

This change would help with https://github.com/ckeditor/ckeditor5-clipboard/pull/30/files#r127750204 – our considerations regarding the fact that dropped content should be inserted a bit differently than pasted content.

Once this is done we may consider dropping the `clipboardInput` event or implementing it differently (perhaps the same change might help as with `inputTransformation`).

**Comment by pjasiun** [comment]
I agree and I'm for dropping `clipboardInput`. This is a common event for `drop` and `paste`. Even if these events seem to be very similar, they are not. Drop needs to calculate the position and always uses collapsed position, when paste should use current selection and remove selected content. For drop we may use some UX helpers, like the magic line. Moreover, after the paste, the position should be after the pasted content. In the case of the drop, the dropped content should be selected. Because of these differences, I think we should not have a common event, like `clipboardInput`.

Of course, some parts are the same, like `inputTransformation` and data insertion, so they should use common API, that's fine. However, the whole process should be handled separately and we should not encourage plugins (like the table plugin) to create a common mechanism to handle both the same way.

**Comment by Reinmar** [comment]
I realised that it would be good to clarify this situation sooner than later because in tests we mock the `clipboardInput` in an inconsistent manner now. Only a few tests provide target ranges because the clipboard feature doesn't use them (since it mostly does pasting now).

So, as soon as we'll want to add dropping support, we'll need also to clean this up.
